src/pages/Home/styled.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+
+const Welcome = styled.div`
+  color: ${({ theme }) => theme.text};
```

Comment:
I've seen this pattern frenquently. You can have it extracted as a general fn as an utility to be reused at several styled component case like:

```js
// at /util/styleUtils.js
export const getThemeField = fieldName => ({ theme }) => theme[fieldName];
export const getThemeText = getThemeField('text');
export const getThemeContent = getThemeField('content');

// Then at this line:
color: ${getThemeText};
// At the line below:
background-color: ${getThemeContent};
```

Reply:
If the code results strange or difficult to follow, check more about [currying](https://medium.com/javascript-scene/curry-and-function-composition-2c208d774983) and decide if keep it or leave it